src/witan/send/vis/general_population.clj

```diff
@@ -36,14 +36,19 @@
    ["General Population Key Stage 5" ay/key-stage-5]
    ["General Population NCY 15+" ay/ncy-15+]
    ["General Population All NCYs" (concat ay/early-years ay/key-stage-1 ay/key-stage-2 ay/key-stage-3 ay/key-stage-4 ay/key-stage-5 ay/ncy-15+)]
-   ["Total General Population" #{99}]])
+   ["Total General Population" #{99}]
+   ;; 99 is used as a placeholder where data structures require a number be present but is
```

Comment:
add `FIXME: ` at the front as I search for that and some modes highlight that for you automagically